sdk/core/Azure.Core.Amqp/src/AmqpAnnotatedMessage.cs

```diff
@@ -98,12 +80,12 @@ public AmqpAnnotatedMessage(IEnumerable<BinaryData> dataBody)
         private Dictionary<string, object>? _messageAnnotations;
 
         /// <summary>
-        /// The properties of the AMQP message.
+        /// Gets the properties of the AMQP message.
```

Comment:
Would call-outs to the spec be helpful here, with respect to answering "what is a property versus and application property?"